pages/common/activator.md

```diff
@@ -0,0 +1,23 @@
+# activator
+
+> Activator aims to be a friendly one-stop-shop to bootstrap your Scala.
```

Comment:
No need to repeat "Activator" again. We can just say "A tool which".

"Scala" should be "Scala projects" ?